File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 3/SOAL 5/PRAK305-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main(){
    int s; scanf("%d", &s);
    int m = (s % 3600) / 60, h = (s % 86400) / 3600, d = s / 86400, detik = s % 60; 
    if (d <= 0){
    printf("%0.2d:%0.2d:%0.2d", h, m, detik);
    } else {printf("%d hari %0.2d:%0.2d:%0.2d", d,  h, m, detik);}
    return 0;
}

File: MODUL 3/SOAL 5/PRAK305-2410817210002-Sebastian_Abe_Santoso.py
s = input(); s = int(s)
m = (s % 3600) // 60; h = (s % 86400) // 3600; d = s // 86400; detik = s % 60; d = int(d)
if d < 1: print(f"{h:02}:{m:02}:{detik:02}")
else: print(f"{d} hari {h:02}:{m:02}:{detik:02}")